### DISCUSSION TITLE
No collective title: when to use space-semicolon-space and when to use period-space

### DISCUSSION THREAD

**Discussion by eblakedc** [rbms-bsc/DCRMR#147] [Graphics]
Our discussion on Friday led me to find another problem. 
(tl;dr: I think DCRM(G) is wrong, and it's such an obscure thing that I don't think it needs to be discussed with the Graphics community. We can just fix it.)

Here's what DCRMR says: 
> [0.2.1.2](https://docs.google.com/document/d/1RxR8ZJ8PCFb9JmrSMz8mlf8hNwNBkP6WT4WjqOnmMZw/edit#bookmark=id.44sinio) For manifestations with no collective title, separate the titles by a space-semicolon-space if the works are all by the same entity, even if the titles are linked by a connecting word or phrase. If the individual works are by different entities _or the authorship is in doubt_, precede each title other than the first by a period and one space, unless a linking word or phrase is already present. Precede each statement of responsibility by a space-slash-space.

At the G meeting on Friday, we concluded that the intention was "If the individual works are for-sure by the same entity, use a semicolon. If the individual works are _or might be_ by different entities, use a period." 

But DCRM(G)'s instructions (1H1) _also_ touch on whether or not there's a shared responsibility statement. The implication in DCRM(G) is that if you have a bunch of titles but just one "/" (because there's only one s.o.r.) you separate with a semicolon; but if you have a bunch of titles and multiple instances of "/" (because different entities were responsible, or because the _same_ statement of responsibility appears with two or more titles) you separate with a period. 

I'm pretty sure that DCRM(G) is just plain wrong. I can easily see how it came about because it was a _very_ last minute change (a few days before the version called "DCRMG for BSC and Exec"). 

Across _years_ of drafting, the examples went like this:

> ### By the same person or body
> [Use space-semicolon-space.]
> 
> > Scene in Printing-House Square, New York City ; Elevated railway in Greenwich Street, New York City / sketched by Stanley Fox
> > (_Comment_: Two images on a single sheet of an illustrated newspaper page, each with a statement of responsibility for the same creator)
>
> > Difficulties of a working general among the bayous ; Bayou navigation in Dixie / from a sketch by Mr. Theodore R. Davis 
> > (_Comment_: Three images share a title and a fourth image has its own title—all on one sheet; same creator)

> ### By different persons or bodies
> [Use period-space.]
> 
> > Charles E. Courtney / photographed by Ernsberger & Ray. Edward Harlan / photographed by J. Bruce & Co.
> > (_Comment_: Two portraits on one sheet, each with a different statement of responsibility)

Then at the last minute, someone must have noticed that the first example failed to account for the fact that each title has an s.o.r. associated with it, yet the s.o.r. was only transcribed once. In the stress of getting DCRM(G) ready for Exec, the omission was over-corrected: in addition to adding the missing s.o.r., the semicolon was changed to a period, and the example got bumped down to the second instruction with the heading changed to "By different persons or bodies, or without a shared statement of responsibility (if any)")

In retrospect, that should have been corrected simply by inserting the missing s.o.r.:
> > Scene in Printing-House Square, New York City / sketched by Stanley Fox ; Elevated railway in Greenwich Street, New York City / sketched by Stanley Fox

Current rules for ISBD Punctuation (in the 2021 Update to the 2011 Consolidated Edition of ISBD) back me up on this: semicolons separate same-creator statements, periods separate different-creator statements, and any statements of responsibility come right after the relevant title. There's no example of two statements of responsibility for the same person in the same language, but it's so specific to late 18th- and 19th-century images that this isn't surprising.

Long story short, I'm going to change it in the draft and link to this discussion topic (because this explanation is waaaaaay too long for a Google Docs comment).

**Comment by jess-grz** [comment]
+1 This makes sense to me. I think you're absolutely right @eblakedc !
